### ISSUE TITLE
Some warnings under Python 3.8.0

### ISSUE THREAD

**Issue by skipik** [alexbers/mtprotoproxy#162]
```
/opt/mtprotoproxy/mtprotoproxy.py:2190: DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.
servers.append(loop.run_until_complete(task))
/opt/mtprotoproxy/mtprotoproxy.py:2195: DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.
servers.append(loop.run_until_complete(task))
An open stream object is being garbage collected; call "stream.close()" explicitly.
```
Could you fix it please?

**Comment by alexbers** [comment]
Ok, the loop argument is removed, https://github.com/alexbers/mtprotoproxy/commit/fd75ca3cf901fab61e6e22b197acf18d26fb02a5. The asyncio is becoming better, I like it.

**Comment by skipik** [comment]
@alexbers thanks!
There's only one extra string now:
`python3.8[1783]: An open stream object is being garbage collected; call "stream.close()" explicitly.`
What does that actually mean? Should I just ignore it?

**Comment by skipik** [comment]
Ah, looks like it's a 3.8 bug: https://www.mail-archive.com/python-bugs-list@python.org/msg395411.html
They are going to fix it in 3.8.1.
https://bugs.python.org/issue38529

**Comment by alexbers** [comment]
Looks like this bug in Python: https://bugs.python.org/issue38529. So you can ignore it. Will likely be fixed by 3.8.1. But I'll try to install Python 3.8 and investigate this a bit

**Comment by skipik** [comment]
Fixed in Python 3.8.1, so I closing it.
